modules/ossm-config-disable-networkpolicy.adoc

```diff
@@ -0,0 +1,40 @@
+////
+This module included in the following assemblies:
+-v2x/servicemesh-release-notes.adoc
+////
+
+[id="ossm-config-disable-networkpolicy_{context}"]
+= Disabling network policies
+
+{ProductName} automatically creates and manages a number of `NetworkPolicies` resources in the control plane and application namespaces. This is to ensure that applications and the control plane can communicate with each other.
+
+If you want to disable the automatic creation and management of `NetworkPolicies` resources, for example to enforce company security policies, you can do so.  You can edit the `ServiceMeshControlPlane` to set the `spec.security.manageNetworkPolicy` setting to `false`
+
+[NOTE]
+====
+When you disable `spec.security.manageNetworkPolicy` {ProductName} will not create *any* `NetworkPolicy` objects.  The system administrator is responsible for managing the network and fixing any issues this might cause.
+====
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Select the project where you installed the control plane, for example `istio-system`, from the Project menu.
+
+. Click the {ProductName} Operator. In the *Istio Service Mesh Control Plane* column, click the name of your `ServiceMeshControlPlane`, for example `basic-install`.
+
+. On the *Create ServiceMeshControlPlane Details* page, click `YAML` to modify your configuration.
+
+. Set the `ServiceMeshControlPlane` field `spec.security.manageNetworkPolicy` to `false`, as shown in this example.
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+spec:
+  security:
+    trust:
```

Comment:
@rh-tokeefe Hi Tim, line 35 .spec security.trust does not match line 28 spec.security.manageNetworkPolicy .
I think you need to remove line 35 trust .

Reply:
or keep the trust indentation as the same layer as manageNetworkPolicy. For example,
https://github.com/jwendell/istio-operator/blob/fe90fa6e46288aeb28b9644dd31a13dd0279d18a/manifests-maistra/2.1.0/servicemeshcontrolplanes.crd.yaml#L4989
The current sample spec `manageNetworkPolicy` should not under `trust`